Sources/Components/Profile/ReputationView.swift

```diff
@@ -89,6 +92,12 @@ public class ReputationView: UIView {
         label.translatesAutoresizingMaskIntoConstraints = false
         label.numberOfLines = 0
         label.textColor = .textAction
+
+        // Make label tapable
+        let gesture = UITapGestureRecognizer(target: self, action: #selector(onLinkTap))
+        label.isUserInteractionEnabled = true
```

Comment:
Maybe move this to be right below the other attributes set on the label?

Reply:
Wow 😍